tt_eager/tt_lib/csrc/tt_lib_bindings_tensor_custom_bmm_ops.cpp

```diff
@@ -122,6 +122,11 @@ namespace tt::tt_metal::detail
             py::arg().noconvert(), py::arg("output_mem_config") = operation::DEFAULT_OUTPUT_MEMORY_CONFIG, R"doc(
             Shuffles [B, 1, S, 4672] fused qkv matrix into 3 heads with shapes [B, 71, S, 64], [B, 1, S, 64], and [B, 1, S, 64].
         )doc");
+        m_tensor.def("nlp_kv_cache_unpad_to_sharded", &nlp_kv_cache_unpad_to_sharded,
```

Comment:
is this doc correct that usually the inputs are `b x nh x s x dh`, or is b in dim1?

Reply:
They are interchangeable so I added both in the latest doc